runtime/src/execute_cost_table.rs

```diff
@@ -45,19 +47,24 @@ impl ExecuteCostTable {
         self.table.len()
     }
 
-    // instead of assigning unknown program with a configured/hard-coded cost
-    // use average or mode function to make a educated guess.
-    pub fn get_average(&self) -> u64 {
+    /// default program cost, set to ComputeBudget::DEFAULT_UNITS
+    pub fn get_default_units(&self) -> u64 {
+        DEFAULT_UNITS as u64
+    }
+
+    /// average cost of all recorded programs
+    pub fn get_average_units(&self) -> u64 {
         if self.table.is_empty() {
-            0
+            self.get_default_units()
         } else {
             self.table.iter().map(|(_, value)| value).sum::<u64>() / self.get_count() as u64
         }
     }
 
-    pub fn get_mode(&self) -> u64 {
+    /// the most frequently occurring program's cost
+    pub fn get_mode_units(&self) -> u64 {
```

Comment:
wdyt about changing these to something like `get_statistical_mode_units`, etc?  For whatever reason I didn't immediately jump to "statistical," but rather, "operating" mode.  I'd have probably misread or been confused by code at the call site